### PR TITLE
IDV POI Submission -- name mismatch

### PR DESCRIPTION
**Pull request by bertrand-deriv**
## IDV POI Submission -- name mismatch [Work in progress]
- setup cypress directory for kyc
- added a custom command
- developed test scenario for name mismatch

### CODE REVIEW

File: cypress/e2e/wip/KYC/idvPoi.cy.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import '@testing-library/cypress/add-commands'
+
+describe('IDV POI submission', () => {
```
Comment (markodonnell-deriv) [inline cypress/e2e/wip/KYC/idvPoi.cy.js:3]: Could you add the test case code? See other tests for example formate.

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Can you exclude the node modules from this PR?

File: cypress/e2e/wip/KYC/idvPoi.cy.js (hunk 1 above)
Comment (bertrand-deriv) [reply cypress/e2e/wip/KYC/idvPoi.cy.js:3]: Yes. it is still in progress

Comment (bertrand-deriv) [reply cypress/e2e/wip/KYC/idvPoi.cy.js:3]: Hi @markodonnell-deriv  It seems someone has merged node modules folder on master 3 weeks ago. The reason why I was having those changes on my PR is that I have pulled the latest master changes into my branch few days ago. I have removed the node modules folder on my branch.

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

File: cypress/e2e/wip/KYC/idvPoi.cy.js (hunk 1 above)
Comment (markodonnell-deriv) [reply cypress/e2e/wip/KYC/idvPoi.cy.js:3]: OK, understood. Please add the test case task id as soon as possible.

**Pull request merged**